deploy/vagrant/conf/alluxio.yml

```diff
@@ -4,13 +4,16 @@
 # {Github | Local | Release}
 Type: Local
 
+# If the version is less than 1.1.0, ie. 1.0.1. This should be false for 1.1.0-SNAPSHOT. {true|false}
+Version_LessThan_1.1.0: false
```

Comment:
Yeah for local it would be tricky, but given that this code is only introduced after 1.0.1, I think it's safe to assume they did not graft a new deploy module onto an older checkout.